src/shopping-cart.js

```diff
@@ -0,0 +1,14 @@
+import cart from './data/order.js';
+import crystals from './data/crystals.js';
+import { getOrderTotal } from './register.js';
+import renderLineItem from './render-line-item.js';
+import toUSD from './format.js';
+
+for(let i = 0; i < cart.length; i++) {
+    const tbody = document.querySelector('tbody');
```

Comment:
I recommend giving the tbody an id in your html file then grabbing it by id instead of using query selector. What if there was another tbody on the html page?